conandata.yml

```diff
@@ -1,7 +1,7 @@
-version: "5.7.0-alpha.1"
+version: "5.7.0-beta.1"
 requirements:
   - "scripta/0.1.0@ultimaker/testing"
 requirements_arcus:
   - "arcus/5.3.1"
 requirements_plugins:
-  - "curaengine_grpc_definitions/(latest)@ultimaker/testing"
+  - "curaengine_grpc_definitions/(latest)@ultimaker/stable"
```

Comment:
I would suggest that you create a Conan package release version and define the pinned version for this in Cura and CuraEngine.